yb-voyager/src/tgtdb/postgres.go

```diff
@@ -1026,3 +1026,90 @@ func (pg *TargetPostgreSQL) getSchemaList() []string {
 	schemas := strings.Split(pg.tconf.Schema, ",")
 	return schemas
 }
+
+func (pg *TargetPostgreSQL) GetEnabledTriggersAndFks() ([]string, error) {
```

Comment:
Again, this also should be limited to tableList. Pls take care of this when you incorporate table lists into guardrails checks.

Reply:
Yep will do it in the table list PR